packages/SwingSet/src/index.js

```diff
@@ -2,12 +2,17 @@ import { loadBasedir, buildVatController } from './controller';
 import { buildMailboxStateMap, buildMailbox } from './devices/mailbox';
 import { buildTimer } from './devices/timer';
 
+import { buildStorageInMemory } from './hostStorage';
```

Comment:
I realize none of these import/export pairs are formatted the way I'd like, but would you be able to change them all to something like:
```js
export { buildStorageInMemory } from './hostStorage';
```

That helps the module machinery figure out when unnecessary imports can be omitted.

Reply:
done in 9afc7b4

It seems more efficient if you go ahead and do that sort of thing. I did check "Allow edits from maintainers."